src/taskpane/server.js

```diff
@@ -1072,61 +1198,135 @@ function contentValidationSetter(sheet, model, fieldTypeEnums, context) {
       case fieldTypeEnums.int:
         setIntegerValidation(sheet, columnNumber, nonHeaderRows, false);
         break;
+      case fieldTypeEnums.customInteger:
+        setIntegerValidation(sheet, columnNumber, nonHeaderRows, false);
+        break;
+      case fieldTypeEnums.customAutomationHost:
+        setIntegerValidation(sheet, columnNumber, nonHeaderRows, false);
+        break;
+
 
       // NUM fields are handled as decimals by Excel though
       case fieldTypeEnums.num:
         setNumberValidation(sheet, columnNumber, nonHeaderRows, false);
         break;
-
-      // BOOL as Sheets has no bool validation, a yes/no dropdown is used
-      case fieldTypeEnums.bool:
-        // 'True' and 'False' don't work as dropdown choices
-        list.push("Yes", "No");
-        setDropdownValidation(sheet, columnNumber, nonHeaderRows, list, false);
+      case fieldTypeEnums.customDecimal:
+        setNumberValidation(sheet, columnNumber, nonHeaderRows, false);
         break;
 
       // DATE fields get date validation
       case fieldTypeEnums.date:
         setDateValidation(sheet, columnNumber, nonHeaderRows, false);
         break;
+      case fieldTypeEnums.customDate:
+        setDateValidation(sheet, columnNumber, nonHeaderRows, false);
+        break;
+      case fieldTypeEnums.customDateAndTime:
+        setDateTimeValidation(sheet, columnNumber, nonHeaderRows, false);
+        break;
+
+      //TEXT types validation

```

Comment:
What you have done is totally fine. For information (no need to change) you could rewrite the below as:
```
case fieldTypeEnums.text:
case fieldTypeEnums.customText:
case fieldTypeEnums.customPassword:
    setTextValidation(sheet, columnNumber, nonHeaderRows, false);
    break;
```

Reply:
Got it! Thanks for the tip